lib/mixins/id.js

```diff
@@ -24,5 +24,11 @@ export default {
         safeId() {
             return this.id || this.localId_ || null;
         }
+    },
+    methods: {
+        safeIdPlus(str) {
+            str = String(str || '').replace(/\s+/g, '_');
+            return (this.safeId && Boolean(str)) ? `${this.safeId}_${str}` : null;
+        }
     }
 };
```

Comment:
IMHO will be more flexible to have one universal methods at this API, which accepts two optional params: `generateId(String suffix = '', Boolean isPrefix=false)`

It could be something like that:
```
generateId(str='', isPrefix=false) {
            str = String(str).replace(/\s+/g, '_');
            if(!!str && this.safeId){
                     return isPrefix? `${str}_${this.safeId}`: `${this.safeId}_${str}`
            }else if(this.safeId){   
            return this.safeId
            }
      return null
}
```